Flashcards.DouglasFir/Flashcards/DAO/FlashCardDao.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Flashcards.Database;
+using Flashcards.Services;
+using Flashcards.DTO;
+using Flashcards.Models;
+using Dapper;
+
+namespace Flashcards.DAO;
+
+public class FlashCardDao
+{
+    private readonly DatabaseContext _dbContext;
+
+    public FlashCardDao(DatabaseContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public void InsertNewFlashCard(FlashCardDto flashCardDto, StackDto stack)
+    {
+        try
+        {
+            FlashCard flashCard = new FlashCard
+            {
+                StackID = stack.StackID,
+                Front = flashCardDto.Front,
+                Back = flashCardDto.Back
+            };
+
+            using (var dbConnection = _dbContext.GetConnectionToFlashCards())
+            {
+                string sql = $"INSERT INTO {ConfigSettings.tbFlashCardsName} (StackID, Front, Back) VALUES (@StackID, @Front, @Back)";
+                dbConnection.Execute(sql, new { flashCard.StackID, flashCard.Front, flashCard.Back });
+            }
+        }
+        catch
+        {
+            throw;
+        }
```

Comment:
In your other code you handle exceptions really well with showing the error to the user for example. Here it's not doing anything. Behaviour will be the same without the try catch.